ShopApp/ShopApp/Controller/HomeViewController.swift

```diff
@@ -70,22 +75,36 @@ class HomeViewController: UIViewController {
     // MARK: - ViewDidLoad
     override func viewDidLoad() {
         super.viewDidLoad()
+        
+        // Set the navigation bar title
+        guard let name = Auth.auth().currentUser?.displayName as? String else { return }
```

Comment:
not sure it's a good idea to use guard here, because if the name will be nil nothing will be executed from viewDidLoad, I think it would be better to use an optional: let name = Auth.auth().currentUser?.displayName ?? ""